rainier-core/src/main/scala/com/stripe/rainier/core/Injection.scala

```diff
@@ -6,19 +6,21 @@ import com.stripe.rainier.unused
 /**
   * Injective transformations
   */
-trait Injection { self =>
+private[rainier] trait Injection { self =>
   def forwards(x: Real): Real
   def backwards(y: Real): Real
   def isDefinedAt(@unused y: Real): Real = Real.one
   def requirements: Set[Real]
-
+  def supportTransform(supp: Support): Support
```

Comment:
can we call this method `transformSupport` instead?

Reply:
Changed